### ISSUE TITLE
Update docs with parameters for each test

### ISSUE THREAD

**Issue by clausherther** [calogica/dbt-expectations#76]
We should list each parameter in the example docs and indicate whether it's required or optional.

Example:

```yaml
tests:
  - dbt_expectations.expect_column_stdev_to_be_between:
      # either min_value or max_value is required 
      min_value: 0 <optional>
      max_value: 2 <optional>
      group_by: [col1, col2] <optional>
      strictly: True <optional>
```

**Comment by makotonium** [comment]
I would love to see these documented and perhaps I could help. If you didn't know to click on each test name link to see its implementation, you wouldn't know that you can use `row_condition`, `group_by_columns` or others on some of the tests that exists. Is just the matter of updating the main README.md file?

By the way, thanks for creating this package! It's been really useful!

**Comment by clausherther** [comment]
Hi @makotonium ! Thanks for your note, I'd love your help on that. Documentation updates are really one of the best contributions to a project like this.
Yes, for now all documentation is in the README.md file, so it would "just" be a matter of updating that. Would you be up for that?

**Comment by makotonium** [comment]
@clausherther Yup. I'll get a draft PR going and ping you once I have some content changes. Thanks for the clarification.

**Comment by clausherther** [comment]
Closed via #124
